tests/update/patch_before_migration.pm

```diff
@@ -66,7 +74,12 @@ sub patching_sle {
 
     }
     else {
-        scc_deregistration(version_variable => 'HDDVERSION');
+        if (sle_version_at_least('12')) {
+            scc_deregistration(version_variable => 'HDDVERSION');
+        }
+        else {
+            assert_script_run('suse_register -E');
+        }
```

Comment:
Same as above, try to wrap up the suse_register for further use.

Reply:
wrap it as sle_register

Reply:
Thanks for the update.

Reply:
Looks more tidy now, thank you